packages/react-components/src/components/MessageThread.tsx

```diff
@@ -67,6 +67,37 @@ const getLatestChatMessage = (messages: (ChatMessage | SystemMessage | CustomMes
   return undefined;
 };
 
+/**
+ * Get the latest participant number from the message array.
+ *
+ * @param messages
+ */
+const getLatestParticipantNum = (messages: (ChatMessage | SystemMessage | CustomMessage)[]): number => {
```

Comment:
Do we still need this? Since we are getting participantNum directly from selector according to what @prprabhu-ms commented

Reply:
removed